docs/usage/shooted_seed.md

```diff
@@ -0,0 +1,73 @@
+# Shooted Seeds
+
+Create shooted seeds with the `shoot.gardener.cloud/use-as-seed` annotation
+
+## Preparation
+
+### Edit `garden` namespace
+
+The namespace "garden" needs 2 new labels for the project:
+
+```yaml
+labels:
+  gardener.cloud/role: project
+  project.gardener.cloud/name: garden
+```
+
+### `garden` Project
+
+The annotation works only for `Shoot`s created in the `garden` namespace. Consequently, we have to create a project for the `garden` namespace first using `kubectl` (we don't use the Gardener Dashboard as it would add a `garden` prefix for the namespace).
+
+Example: [/example/05-project-dev.yaml](../../example/05-project-dev.yaml)
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Project
+metadata:
+  name: garden
+spec:
+  owner:
+...
+  namespace: garden
+```
+
+## Create shoot
+
+### Deploy shoot
+
+Change the networks for the shooted seed that it is not overlapping with his shoots (spec.provider.infrastructureConfig.networks and spec.networking).
```

Comment:
How can you "change"  the network of the shooted seed that is not created yet?
I guess that your intention is to create a `Seed` via shooted Seed,  that can have the same network ranges as Shoots running on the garden/root cluster. As such, this is an optional configuration.

If that is the intention of that section,  I would reformulate to make it a bit more clear.

Reply:
Can you recheck this section?